ext/recaptcha/settings/recaptcha.setting.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ * Settings metadata file
+ */
+return [
+  'recaptchaPublicKey' => [
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'recaptchaPublicKey',
+    'type' => 'String',
+    'quick_form_type' => 'Element',
+    'html_attributes' => [
+      'size' => 64,
+      'maxlength' => 64,
+    ],
+    'html_type' => 'text',
+    'default' => NULL,
+    'add' => '4.3',
+    'title' => ts('reCAPTCHA Site Key'),
```

Comment:
Probably should be updated to be E::ts()

Reply:
Thanks @seamuslee001 updated!